Pylint

fretchen opened the pull request:
I finished pylinting and adapted all the notebooks.

fretchen committed:
pylint the pennylane_ls folder

fretchen committed:
Pylint the tests too.

fretchen committed:
And also make sure that the examples are running properly

RohitPrasadBhatt reviewed (changes requested):
My comments:
- The notebooks are not displaying output for each cell and some even have error output. Is this intentional?
- In the ``__init__.py`` we are not importing multiqudit observables. Is this OK? Also the fermion Load operation is not being imported.

fretchen committed:
Update the notebooks and remove the old ops

The ops where poorly overloaded anyways and are not tested. These imports might come back, when they are properly tested.

RohitPrasadBhatt reviewed (approved):
LGTM!
